dispatcher_plugin_nb2workflow/products.py

```diff
@@ -45,7 +46,7 @@ def __init__(self, encoded_data, data_product_type = BaseQueryProduct, out_dir =
                                                      data= numpy_data_prod,
                                                      meta_data=metadata,
                                                      file_dir = out_dir,
-                                                     file_name = f"{self.name}")
+                                                     file_name = f"{self.name}.fits")
```

Comment:
always fits?

Reply:
By default, yes. The constructor in this base class is for NumpyDataProduct based products. 
In other types of product, the constructors are overriden.